src/components/select-project/select-project.controller.ts

```diff
@@ -94,7 +106,7 @@ export class SelectProjectController implements angular.IController {
     };
 
     var filteredProjects = _.reject(projects, 'metadata.deletionTimestamp');
-    this.ctrl.projects = _.sortBy(filteredProjects, this.$filter('displayName'));
+    this.ctrl.projects = this.RecentlyViewedProjectsService.orderByMostRecentlyViewed(_.toArray(projects));
```

Comment:
We probably want to update `orderByMostRecentlyViewed` to call `_.toArray` when needed, but OK as a follow on.